sdk/eventhub/event-hubs/src/eventHubClient.ts

```diff
@@ -351,6 +355,7 @@ export class EventHubClient {
    * @throws {AbortError} Thrown if the operation is cancelled via the abortSignal.
    */
   async getPartitionProperties(partitionId: string, abortSignal?: AbortSignalLike): Promise<PartitionProperties> {
+    throwErrorIfConnectionClosed(this._context);
```

Comment:
Should we add a @throws section to the JSDOC for this error? Maybe add something like "Create a new EventHubsClient and call getPartitionProperties when encountering this error"